README.md

```diff
@@ -321,27 +321,27 @@ via a unique URL.
 
 #### Reference set capture
 Before visual regression testing can begin, reference screenshots need to be captured.
-A reference set can be captured by this command in the `node_modules/backstop_data/` directory:
+A reference set can be captured by this command in the `node_modules/backstopjs/` directory:
 ```
 npm run reference
 ```
 After the reference set capture is finished, then tests can be run against the reference set.
 Currently in `backstop.json` there are reference capture scenarios for:
-- N&C Homepage
+- N&amp;C Homepage
```

Comment:
I think Markdown will turn this into an ampersand, but I didn't think using the HTML entity was necessary in Markdown.